### PR TITLE
Test day 2

### PR DESCRIPTION
**Pull request by nejidevelops**
## In this PR, we worked so hard to achieve the following:
- [x]  Use jest framework for testing.
- [x]  Create a test file(s) ([..].test.js) for a file(s) containing the following functions:
- a function for editing the task description.
- a function for updating an item's 'completed' status.
- the "Clear all completed" function.

### CODE REVIEW

File: src/add-task.test.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+/* eslint-disable prefer-const */
+/* eslint-disable no-unused-vars */
```
Comment (nahnahdev) [inline src/add-task.test.js]: - [ ] Great job so far @NewtonOmbese and @tan12082001 There are just a few changes I will suggest you implement and you'll be on your way to the next milestone :+1: 
- [ ] I noticed that you used a lot of `eslint disable` commands in your project. I recommend avoiding the use of `eslint disable commands` in your projects, as these errors and warnings are typically helpful for identifying areas where you can improve your code. By resolving these issues, you can write more clean and readable code that adheres to industry standards, which can make it easier to maintain your codebase over time. :100:
- [ ] So kindly remove all the `eslint disable` commands in your project :+1:

File: src/clearall-checked.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable import/named */
+/* eslint-disable import/prefer-default-export */
+// import { updateTaskIndexes } from './index.js';
```
Comment (nahnahdev) [inline src/clearall-checked.js]: - [ ] Kindly remove all the `eslint disable` commands in your project :+1:

File: src/clearall-checked.test.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable global-require */
+
```
Comment (nahnahdev) [inline src/clearall-checked.test.js]: - [ ] Kindly remove all the `eslint disable` commands in your project :+1:

File: src/update-taskinput.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+/* eslint-disable import/prefer-default-export */
```
Comment (nahnahdev) [inline src/update-taskinput.js]: - [ ] Kindly remove all the `eslint disable` commands in your project :+1:

File: src/update-taskinput.test.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable global-require */
+
```
Comment (nahnahdev) [inline src/update-taskinput.test.js]: - [ ] Kindly remove all the `eslint disable` commands in your project :+1:

File: src/updateTaskStatus.js [hunk 6]
```diff
@@ -0,0 +1,7 @@
+/* eslint-disable import/prefer-default-export */
```
Comment (nahnahdev) [inline src/updateTaskStatus.js]: - [ ] Kindly remove all the `eslint disable` commands in your project :+1:

File: src/deleteTask.test.js [hunk 7]
```diff
@@ -43,8 +42,8 @@ describe('deleteTask', () => {
 
     deleteTask(taskToDelete, tasks);
 
-    expect(jest.spyOn(indexModule, 'updateTaskIndexes')).toHaveBeenCalled();
-    expect(jest.spyOn(indexModule, 'saveTasks')).toHaveBeenCalled();
+    // expect(jest.spyOn(indexModule, 'updateTaskIndexes')).toHaveBeenCalled();
+    // expect(jest.spyOn(indexModule, 'saveTasks')).toHaveBeenCalled();
```
Comment (nahnahdev) [inline src/deleteTask.test.js]: - [ ] Kindly avoid leaving commented pieces of code in your codebase. It makes your codebase unclean and is not a good practice :+1:

File: src/index.js [hunk 8]
```diff
@@ -1,33 +1,20 @@
+/* eslint-disable import/named */
```
Comment (nahnahdev) [inline src/index.js]: - [ ] Kindly remove all the `eslint disable` commands in your project :+1:

**Review by nahnahdev — CHANGES_REQUESTED** [review]
## Project Status: CHANGES REQUIRED :recycle: 
Hi @NewtonOmbese and @tan12082001 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :fire: 
- All required tests present and passing :heavy_check_mark: 
- Linters well setup :heavy_check_mark: 
- Professional README :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @NewtonOmbese , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All linters passed

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @TEAM

Your project is complete! Great job for naking the changes requested 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Works as expected
- Test well added
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**